XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -501,7 +508,17 @@ private void guestMetrics_PropertyChanged(object sender, PropertyChangedEventArg
         {
             if (e.PropertyName == "other")
             {
+                if (RDPEnabled)
+                {
+                    if (vncScreen.UseVNC && (tryToConnectRDP || (!vncScreen.UserWantsToSwitchProtocol && Properties.Settings.Default.AutoSwitchToRDP)))

```

Comment:
Ah yes, thank you, I understand now. I was confused because I thought that UserWantsToSwitchProtocol was a new variable in your changeset, but in fact it was there previously.